psalm.xml

```diff
@@ -1,19 +1,21 @@
 <?xml version="1.0"?>
-<psalm errorLevel="4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="https://getpsalm.org/schema/config" xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd" errorBaseline="tests/psalm-baseline.xml">
+<psalm errorLevel="4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="https://getpsalm.org/schema/config"
+	xsi:schemaLocation="https://getpsalm.org/schema/config vendor-bin/psalm/vendor/vimeo/psalm/config.xsd"
```

Comment:
Now you are using `psalm.phar`, so this file does not exist anymore. Meaning you should see a XML error within this file now. So now it is:

```suggestion
	xsi:schemaLocation="https://getpsalm.org/schema/config https://getpsalm.org/schema/config"
```